Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Themes/InteractableThemeProperty.cs

```diff
@@ -16,20 +17,43 @@ public class InteractableThemeProperty
         public InteractableThemePropertyValueTypes Type;
         public List<InteractableThemePropertyValue> Values;
         public InteractableThemePropertyValue StartValue;
-        public int PropId;
+        public int PropId; // i.e OptionIndex
         public List<ShaderProperties> ShaderOptions;
         public List<string> ShaderOptionNames;
         public InteractableThemePropertyValue Default;
         public string ShaderName;
 
-        public string GetShaderPropId()
```

Comment:
@killerantz do you think that the chances are fairly low that someone else would be using this outside of the interactable family? This is technically a breaking change (in this case if we have any thoughts of there being usage we might keep this around and just mark it obsolete - or just redirect it to the renamed thing).

Assuming it's fairly low it seems reasonable to do things this way, just wanted to confirm

Reply:
As far as I know they are only used in Interactable and this is pretty deep. Just make sure it's documented well, in the off chance someone is using it, they can easily update their code to store an int instead of string.

Reply:
Added documentation in code and will add documentation to github page